dspace-api/src/main/java/org/dspace/authority/PersonAuthorityValue.java

```diff
@@ -140,8 +140,8 @@ public SolrInputDocument getSolrInputDocument() {
     @Override
     public void setValues(SolrDocument document) {
         super.setValues(document);
-        this.firstName = ObjectUtils.toString(document.getFieldValue("first_name"));
-        this.lastName = ObjectUtils.toString(document.getFieldValue("last_name"));
+        this.firstName = Objects.toString(document.getFieldValue("first_name"));
```

Comment:
ObjectUtils.toString(null) returns "" while Objects.toString(null) returns "null.  OK?  If not, Objects.toString(thing, "") should be an exact replacement.

Reply:
This has been fixed now in https://github.com/DSpace/DSpace/pull/2669/commits/280ccac20cbd93a187e315dada42c5f55f2a710b